admin-ui/src/pages/settings-page.tsx

```
@@ -0,0 +1,697 @@
+            )}
+            <div className="text-muted-foreground text-xs">
+              Controls which model receives lightweight/free requests when routing.
+            </div>
+          </div>
+
+          <div className="grid gap-2">
+            <Label className="text-muted-foreground text-xs">API key</Label>
+            <Input
+              placeholder="sk-..."
+              value={draft.apiKey ?? ""}
+              onChange={(e) => setDraft((prev) => ({ ...prev, apiKey: e.target.value }))}
```

ESLint

[error] Replace `·setDraft((prev)·=>·({·...prev,·apiKey:·e.target.value·}))` with `⏎················setDraft((prev)·=>·({·...prev,·apiKey:·e.target.value·}))⏎··············`

(prettier/prettier)